CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+- Added capability of settings Save, Division and Shortcut elements of Initial MIDI objects https://github.com/GrandOrgue/grandorgue/issues/1974
```

Comment:
Perhaps it would be better phrased as:

```suggestion
- Added capability of setting Send, Division and Shortcut events for Initial MIDI objects https://github.com/GrandOrgue/grandorgue/issues/1974
```